docs/code-samples/getting-started/daml/Scripts/Holding.daml

```diff
@@ -1,6 +1,6 @@
 module Scripts.Holding where
 
-import DA.Map (empty, fromList)
+import DA.Map qualified as M (empty, fromList)
```

Comment:
Should we use qualified as `Map` and `Set`, as Luc suggested?

Reply:
Yes, we could do that in the tutorials and getting started. First wanted to do the same as we do in the library, but will change that.